xml/dm_registers.xml

```diff
@@ -276,9 +276,19 @@
 
     <register name="Abstract Control and Status" short="abstractcs" address="0x16">
         <field name="0" bits="31:29" access="R" reset="0" />
-        <field name="progsize" bits="28:24" access="R" reset="Preset">
+        <field name="progbufsize" bits="28:24" access="R" reset="Preset">
             Size of the Program Buffer, in 32-bit words. Valid sizes are 0 - 16.
 
+            0: Tiny systems that support System Bus Access might not have a
+            Program Buffer at all.
+
+            1: Tiny systems can support just a single-word program buffer. The
```

Comment:
Putting these here makes it sound like the only options. I think this explanatory text belongs where program buffer is discussed (despite the fact that the TODO Is here).